src/goto-instrument/contracts/contracts.cpp

```diff
@@ -665,7 +665,7 @@ void code_contractst::instrument_assign_statement(
   goto_programt::instructionst::iterator &instruction_iterator,
   goto_programt &program,
   exprt &assigns,
-  std::set<irep_idt> &freely_assignable_symbols,
+  std::set<irep_idt> &local_assignable_symbols,
```

Comment:
I think the old name was more intuitive. Freely assignable would indicate that these things can be assigned to without any checks.

When we insert local variables into this set, we can leave a comment that local variables are always freely assignable.

Reply:
Implemented as suggested.